### ISSUE TITLE
ApteSE-HeadWordCorrections

### ISSUE THREAD

**Issue by Shalu411** [sanskrit-lexicon/CORRECTIONS#117]
Namaste
@funderburkjim I have already given the link- Here I give again-
https://www.dropbox.com/s/yqnuxqdcdi61nbk/AP.txt?dl=0
Please see at line 85 and then move above.

I don't understand the 'strangeness' in L=25285 example.
भृङ्गिरि [L=25285] [p= 1209-2] भृङ्गिरि (री) टिः See भृङ्गरिटि. [Page1210-1]
भृङ्गिरि (री) टिः -- this should have been the original head word. But it does not appear so in the list of head words. Only भृङ्गिरि this part appears. And then when we click the article, we see भृङ्गिरि (री) टिः.
Many such cases are there till now. Please see the text file for others. [Please see at line 85 and then move above. ]

**Comment by gasyoun** [comment]
What @Shalu411 wanted to say that in the headword list it should be भृङ्गिरिटिः or भृङ्गिरीटिः. Now it's wrong, because only the first part is taken until `(`.

**Comment by funderburkjim** [comment]
@gasyoun Your comment was helpful .  It is partly a 'multiple-headword' problem  (like an MW 'OR' case),
and partly an error in the digitization.

Here's the underlying record in ap.txt:
```
.{#BfNgiri#}¦ ({#{@rI@}#}) {#{@wiH@}#} See {#BfNgariwi#}. [Page1210-1]
```
I propose to change to TWO records:
```
.{#BfNgiriwiH#}¦ See {#BfNgariwi#}. [Page1210-1]
.{#BfNgirIwiH#}¦ See {#BfNgariwi#}. [Page1210-1]
```

Agree?

@Shalu411   DO mention such 'strange' cases when you notice them :+1: --- there usually is something wrong
somewhere.

**Comment by gasyoun** [comment]
I agree with two records if we note them as such, that they are digital and nobody will ever find them in book in the exact form. Jim, have you seen the other cases in the AP.txt file? I wonder if there is a way other than listing all the pages of the book to find other similar `()` cases. Actually as @Shalu411 is quick with Devanagari it might be a good idea to look through all pages of AP in search for the undocumented, non o vs O cases of multiple headwords lost. @Shalu411 what's your take?

**Comment by Shalu411** [comment]
@funderburkjim 
``It is partly a 'multiple-headword' problem (like an MW 'OR' case), and partly an error in the digitization.``
Oh, saved. When digitized data has it in tact, as you show, then, can a new list be generated with proper complete multiple head words, solved? Because many cases till now, are of that category, and putting that long comment every time is time taking and will be of no use practically. 
All strange words, I click for, have similar problem. 
@gasyoun 
Sure. So issue inside issue now. :) All 35 such Htmls might need same re-looking?

``@Shalu411 DO mention such 'strange' cases when you notice them :+1: --- there usually is something wrong somewhere.``
So let me give you right now- which are done so far. 
8/26 cases of double/multiple word category, where head word appears partial, incomplete.
----------------------------------
निष्ठी->निष्ठी(ष्ठे)वः ;-ch This is a double word. On opening bracket it will give निष्ठीवः निष्ठेवः  But in the digitizing, they made entry of only निष्ठी  out of निष्ठी(ष्ठे)वः. 
परथा->परता
पारश->पारश(स)व, पारश(स)वी ;-ch This is a double double word. On opening bracket it will give पारशव, पारसव पारशवी, पारसवी. But in the digitizing, they made entry of only पारश out of पारश(स)व, पारश(स)वी.
पारिया->पारिया(पा)त्रः ;-ch 1) This is a double word. On opening bracket it will give पारियात्रः पारिपात्रः. But in the digitizing, they made entry of only पारिया out of पारिया(पा)त्रः. 2) Next there are two words with same base. पारिया(पा)त्रिकः. which open as पारियात्रिकः पारिपात्रिकः.
परीरम्->परिरम्
पेर->पेर(रो)जम् ;-ch This is a double word. On opening bracket it will give पेरजम्, पेरोजम्. But in the digitizing, they made entry of only पेर out of पेर(रो)जम्.
पोथा->पोता
प्रतिघा->प्रतिगा
प्रामोद->प्रामोद(दि)क(की) ;-ch  This is a double double word. On opening bracket it will give प्रामोदक, प्रामोदिक, प्रामोदकी, प्रामोदिकी. But in the digitizing, they made entry of only प्रामोद out of प्रामोद(दि)क(की). 
प्रवेणीः->प्रवेणिः(णी) ;-ch 
फालिका->फलिका
भृङ्गिरि->भृङ्गिरि(री)टिः ;-ch This is a double word. On opening bracket it will give भृङ्गिरिटिः, भृङ्गिरीटिः. But in the digitizing, they made entry of only भृङ्गिरि out of भृङ्गिरि(री)टिः.  
वाधु->वाधु(धू)क्यम् ;-ch This is a double word. On opening bracket it will give वाधुक्यम्, वाधूक्यम्. But in the digitizing, they made entry of only वाधु out of वाधु(धू)क्यम्. 
वर्धकी->वर्धकि
वातूली->वातूली-भ्रमः ;-ch Word वातुलिः is also there
विध्र->विद्र
विश्रुती->विश्रुति
व्याकुलयाति->व्याकुलयति
शाकुल->शकुल
शार्वरीक->शर्वरीक
शार्वरीक->शार्वरिक
साराथिः->सारथिः ;-ch
सूनावत्->सूनवत्
सैरी->सैरि
स्थक->स्थक(ग)रम् ;-ch This is a double word. On opening bracket it will give स्थकरम्, स्थगरम्. But in the digitizing, they made entry of only स्थक out of स्थक(ग)रम्.

**Comment by gasyoun** [comment]
@Shalu411 All 35 such Htmls might need same re-looking? - no, of course. First of all there are major primary and secondary dictionaries. Apte is primary and that's why I would go for it. Not all use such method. We'll need to make a list of dictionaries using `()`.
When you quote I would ask you to use ``` above and ``` bellow the quote. And if you quote double-words, than leave the rest out, otherwise a mess, thanks.

**Comment by Shalu411** [comment]
@gasyoun Got it. sure

**Comment by funderburkjim** [comment]
I've tried to analyze all the words from your ap.txt from the point of view of double words.  
The current count is 87 such cases.
I hope to have the analysis of these cases completed soon.

**Comment by funderburkjim** [comment]
An analysis of the double headwords has now been done.  There ended up 85 cases.
There are slp1 and Devanagari versions [here](https://gist.github.com/funderburkjim/fa50699bfc4a527e804b).

Here's an example of how to read:

7 18962 अम्लि -> अम्लिका,अम्लीका : `#1` अम्लि(म्ली)का

*  7 = the line number within Shalu's AP.txt file referenced above
* 18962 = the line number within the full Cologne digitization of AP 
*  अम्लि = the current headword spelling
*  अम्लिका,अम्लीका = the revised (corrected) headword spellings, (two alternates in this case)
* `#1`  = a method identifier  (`#1` and `#2` were generated by a program; `#3` cases were done manually)
* अम्लि(म्ली)का =  the (probable) way the headword appears in the scan -   this is inferred, since the digitization of AP does not give this directly.

@Shalu411   Would you check that the alternates shown are accurate?

**Comment by Shalu411** [comment]
Namaste
``Would you check that the alternates shown are accurate?``
Sure. I was waiting for this. Now I can look at the Html again with peace.. :) Thanks.

**Comment by drdhaval2785** [comment]
@Shalu411 Any dent ?
